### ISSUE TITLE
Request - add pretrained model

### ISSUE THREAD

**Issue by ofirkris** [thomasjhuang/deep-learning-for-document-dewarping#5]
Hi, thanks for sharing this great project.
Can you share a pre-trained model so we can evaluate easily?

**Comment by thomasjhuang** [comment]
Sure, Ill try and add pre-trained model to update the repo. There is another issue regarding data loading which I will also need to resolve

**Comment by ofirkris** [comment]
Thanks

**Comment by ofirkris** [comment]
Hi Thomas, can you share your weights? Thanks!

**Comment by thomasjhuang** [comment]
At the moment I am quite busy with a new job of mine, and I just don't have the time at the moment to be able to share weights on this as this was a project I worked on a while ago. I will accept anyone who is willing to share a pretrained model. I don't have compute resources at home for me to use, nor am I going to use my company's resources on my personal project here. Before, when I was working on this, I had compute resources provided to me by an organization I was working in private with, and I cannot share their data. What is exposed here is not apart of the NDA I had signed.
